docs/notebooks/entropic_map_estimator.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.    entpots = EntropicPotentials(out.f, out.g, out.geom,a,b)
</pre><p>You can use <code>out.to_dual_potentials</code> to create the <code>EntropicPotentials</code> object.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>